config/services.yaml

```diff
@@ -7,6 +7,7 @@ parameters:
     default_database_driver: "mysql"
     default_admin_auth_bypass: "false"
     timezone: '%env(APP_TIMEZONE)%'
+    public_calendar_enabled: '%env(bool:PUBLIC_CALENDAR_ENABLED)%'
```

Comment:
Since public calendars are enabled for now, we should set "True" as the default value if the env is not set. Probably with something like `env(default:fallback_param:bool:PUBLIC_CALENDAR_ENABLED)` setting `fallback_param` to True (with a better naming of course)